sched/misc/assert.c

```diff
@@ -610,7 +623,7 @@ static void pause_all_cpu(void)
   int delay = CONFIG_ASSERT_PAUSE_CPU_TIMEOUT;
 
   CPU_CLR(this_cpu(), &cpus);
-  nxsched_smp_call(cpus, pause_cpu_handler, NULL, false);
+  nxsched_smp_call_async(cpus, &g_call_data);
```

Comment:
Should you allocate g_call_data locally instead ?

Reply:
"nxsched_smp_call_async" is asynchronous, and local variables will be freed, leading to errors.

Reply:
Can you explain how g_call_data doesn't get clobbered by the same / a different CPU before all have handled it?

Reply:
line 631-643 will wait all other cpu stop